cmd/check-gh-automation/main.go

```diff
@@ -39,6 +39,7 @@ type options struct {
 	releaseRepoPath string
 	flagutil.GitHubOptions
 	pluginConfig prowpluginconfig.PluginOptions
+	adminBots    flagutil.Strings
```

Comment:
How do we plan on using this? is `openshift-merge-robot` going to be the only bot added here?

Reply:
openshift-merge-robot is the only bot being added here but it will be added as a value passed from hack/local-check-gh-automation.sh where it is stored as a hardcoded value using the --adminBots flag. Seemed like the most logical approach but I'm open to suggestions. Using the pre existing --bot was another option but it also included openshift-ci-robot which was outside of the scope